compiler/incremental-compilation-impl/src/org/jetbrains/kotlin/incremental/classpathDiff/ClasspathSnapshotSerializer.kt

```diff
@@ -58,13 +62,13 @@ open class DataExternalizerForSealedClass<T>(
     override fun save(output: DataOutput, objectToExternalize: T) {
         val inheritorClassIndex =
             inheritorClasses.indexOfFirst { it.isAssignableFrom(objectToExternalize!!::class.java) }.also { check(it != -1) }
-        output.writeInt(inheritorClassIndex)
+        output.writeByte(inheritorClassIndex) // Write byte so the data is smaller
```

Comment:
Should we assert that it fits into a byte?

Reply:
Yes ifmt should fit and function together

Reply:
I good job

Reply:
> Should we assert that it fits into a byte?

Done